tests/test_pipeline.sh

```diff
@@ -5,10 +5,22 @@ set -Eeuo pipefail
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 SOURCE_DIR=$(dirname $SCRIPT_DIR)
 
+# Build the Source VCF
+cat << EOT > "${SCRIPT_DIR}/resources/source.vcf"
+##fileformat=VCFv4.2
```

Comment:
```suggestion
##fileformat=VCFv4.3
```

Since it's the latest version (and it should still be valid against 4.3 as far as I can see)